frontend/src/pages/Landing/Home/LandingPage.tsx

```diff
@@ -210,51 +233,83 @@ export const LandingPage = (): JSX.Element => {
         </Stack>
       </LandingSection>
       <LandingSection>
-        <SectionTitleText>Used by most government agencies</SectionTitleText>
+        <SectionTitleText>
+          {t('features.landingPage.usedByAgenciesSection.title')}
+        </SectionTitleText>
         <Wrap shouldWrapChildren spacingX="3rem" mt="2.5rem" spacingY="2.5rem">
-          <StatsItem stat={data?.formCount} description="forms launched" />
+          <StatsItem
+            stat={data?.formCount}
+            description={t(
+              'features.landingPage.usedByAgenciesSection.formsLaunched',
+            )}
+          />
           <StatsItem
             stat={data?.submissionCount}
-            description="submissions received"
+            description={t(
+              'features.landingPage.usedByAgenciesSection.submissionsReceived',
+            )}
           />
           <StatsItem
             stat={data?.userCount}
-            description="public officers onboard"
+            description={t(
+              'features.landingPage.usedByAgenciesSection.publicOfficersOnboard',
+            )}
           />
           <StatsItem
             stat={data?.agencyCount}
-            description="government agencies"
+            description={t(
+              'features.landingPage.usedByAgenciesSection.governmentAgencies',
+            )}
           />
         </Wrap>
-        <VisuallyHidden>Examples of users of FormSG</VisuallyHidden>
+        <VisuallyHidden>
+          {t('features.landingPage.usedByAgenciesSection.exampleUserTitle')}
+        </VisuallyHidden>
         <Wrap
           shouldWrapChildren
           mt="4rem"
           spacing={{ base: '2.5rem', md: '4.5rem' }}
         >
           <Image
-            alt="Immigration and Checkpoints Authority"
```

Comment:
I feel that these shouldn't be translated since it is so specific to Singapore 🫣

Reply:
oo yea agree. 
a rule of thumb then I think: extract texts like `features.landingPage.something.not.country.specific` instead of something super specific like `features.landingPage.usedByAgenciesSection.imageAlt.mom`

lemme do the change in a bit

Reply:
@KenLSM revised!